server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputationSteps.java

```diff
@@ -41,6 +41,7 @@
       ParseReportStep.class,
 
       // Persist data
+      ComputeAndPersistDaysSinceLastCommitStep.class,
```

Comment:
For the time being I'm not sure it makes sense to have these both verbs, but as you wish, we'll change that when working in the "Decorator" sprint.